changelog/unreleased/example-deployments.md

```diff
@@ -0,0 +1,6 @@
+Change: disable the color logging in docker compose examples
```

Comment:
I wouldn't consider this to be a breaking change, thus I'd expect `Enhancement`. 🤔

Reply:
`Change` is supposed to mean breaking change? I wasn't aware of that. I always used change for refactorings or stuff which didn't change the functionality.

Reply:
fooled me as well for a long time. But in calens, the three categories map 1:1 to the three version parts in semver:
- `Bugfix` = patch version (third section)
- `Enhancement` = minor version (second section)
- `Change` = breaking = major version (first section)
Doesn't have any other semantics than that.